ls1intum/Artemis
# `Assessment`: Enable save button on all changes in modeling assessments

### Checklist
#### General


#### Client

### Motivation and Context


Even with the fix from https://github.com/ls1intum/Apollon/pull/295, the save button is sometimes disabled when it should not be.

### Description

The problem is that the disabled state is not refreshed often enough. A single `return` statement prevented the state to be updated when the Apollon editor somehow was not loaded in the background. Removing this `return` fixes the problem.

### Steps for Testing

Prerequisites:
- 1 exam with >= 1 modeling exercise and a participation (e.g. https://artemis-test5.artemis.cit.tum.de/course-management/9/exams/11)
- 1 tutor to assess the exam

1. Go to the exam assessment dashboard
2. Assess the modeling exercise
3. Add feedback to a part of the submission (referenced feedback). Don't add any text, just the credits.
4. Observe that the save button is enabled. Before, it was still disabled (see https://github.com/ls1intum/Apollon/pull/295#pullrequestreview-1530490008)

#### Exam Mode Testing


see above

### Review Progress






#### Performance Review
#### Code Review
#### Exam Mode Test

### Test Coverage







### Screenshots



https://github.com/ls1intum/Artemis/assets/9006596/5db08f57-d329-4b62-ba5e-177b19fbf81b
